Gemfile

```diff
@@ -65,6 +65,7 @@ group :cable do
   gem 'puma', require: false
 
   gem 'em-hiredis', require: false
+  gem 'hiredis', require: false
```

Comment:
It's used [here](https://github.com/rails/rails/pull/23381/files#diff-5e21544fd19a4d670c1c6801418b892fR12) to ensure we work with that driver as well.